vizro-core/docs/pages/examples/examples.md

```diff
@@ -23,3 +23,7 @@ The [`examples/features` folder](https://github.com/mckinsey/vizro/tree/main/viz
 !!! info "Ways to configure a dashboard"
 
     In most cases, the pydantic model should be your preferred method of dashboard configuration, but you can also [define a dashboard with YAML, JSON, or as a Python dictionary](../user-guides/dashboard.md).
+
+## Examples from Vizro users
+
+We maintain a separate, curated page of [videos, blog posts, and examples of Vizro usage from our community](./your-examples.md).
```

Comment:
```suggestion
We maintain a separate, curated page of [videos, blog posts, and examples of Vizro usage from our community](your-examples.md).
```

Very minor, has no real impact on anything, but just makes things a bit simpler. It seems like we always do link this way apart from two remaining `./` style links in `first-dashboard.md`, so if you can be bothered then please can you make the change there too?